batch/batch/cloud/terra/azure/worker/worker_api.py

```diff
@@ -0,0 +1,95 @@
+import os
+from typing import Dict, List
+
+from hailtop import httpx
+from hailtop.aiocloud.aioazure import AzureCredentials
+from hailtop.aiocloud.aioterra.azure import TerraAzureAsyncFS
+from hailtop.aiotools.fs import AsyncFS
+from hailtop.auth.auth import IdentityProvider
+
+from .....worker.credentials import CloudUserCredentials
+from .....worker.disk import CloudDisk
+from .....worker.worker_api import CloudWorkerAPI, ContainerRegistryCredentials
+from ....terra.azure.instance_config import TerraAzureSlimInstanceConfig
+
+
+class AzureManagedIdentityCredentials(CloudUserCredentials):
+    @property
+    def cloud_env_name(self) -> str:
+        return 'HAIL_IGNORE'
+
+    @property
+    def mount_path(self) -> str:
+        return ''
+
+    @property
+    def identity_provider_json(self):
+        return {'idp': IdentityProvider.MICROSOFT.value}
+
+
+class TerraAzureWorkerAPI(CloudWorkerAPI[AzureManagedIdentityCredentials]):
+    nameserver_ip = '168.63.129.16'
```

Comment:
unused perhaps?

Reply:
This is a class variable that is used by `worker.py`. It probably is inconsequential in terra but I copied this value from the Azure worker API.